hirundo/storage.py

```diff
@@ -233,15 +329,21 @@ def delete(self) -> None:
             raise ValueError("No StorageIntegration has been created")
         self.delete_by_id(self.id)
 
-    def create(self) -> int:
+    def create(self, replace_if_exists: bool = False) -> int:
```

Comment:
you return int. This is the ID?

Reply:
Maybe write it as create_id? I thought it was the creation of an object from this class

Reply:
It does create an object from the class. It creates a `dataset` in the DB on the server and returns the ID of that new `dataset` instance. Is there some way that I can make this clearer?

Reply:
i see. so you could add a short documentation. I think this makes sense